### PR TITLE
exclude body

### PR DESCRIPTION
**Pull request by galcohen92**
add exclude body in request and response, when default is false.

### CODE REVIEW

File: lib/logger-helper.js [hunk 1]
```diff
@@ -57,98 +60,100 @@ var auditResponse = function(req, res, options){
...
     var requestRoute = utils.getRoute(req);
     var queryParams = req && req.query !== {} ? req.query : NA;
     var method = req && req.method ? req.method : NA;
     var URLParams = req && req.params ? req.params : NA;
     var timestamp = req && req.timestamp ? req.timestamp.toISOString() : NA;
     var timestamp_ms = req && req.timestamp ? req.timestamp.valueOf() : NA;
+    var requestBody =  _.get(req, 'body');  //handle body clone the original body
 
-    var requestBody;
+    requestBody = handleJson(requestBody, options.logger, options.request.excludeBody, options.request.maskBody);
```
Comment (kobik) [inline lib/logger-helper.js]: Why not calling it like that?

`requestBody = handleJson(requestBody, options);`

File: lib/utils.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 var _ = require('lodash');
 const MASK = 'XXXXX';
+var ALL_BODY = '*';
```
Comment (kobik) [inline lib/utils.js]: Rename `ALL_BODY` to something more generic, as it's now relevant also for headers

File: lib/logger-helper.js (hunk 1 above)
Comment (galcohen92) [reply lib/logger-helper.js]: because I call it in the responseAudit as well - excludebody and maskBody are under options.response this time

Comment (galcohen92) [reply lib/logger-helper.js]: and when you call handleJson for headers or body the field from options are diff:
1. headers -  excludHeaders and maskHeaders
2. body - excludBody and maskBody

**Pull request merged**